api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1139,101 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Creating a subscription user subscribes to event-types and
+      later can read this subscription with many streams.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'use_case'.
+      It's not possible to have two different subscriptions with these properties being the same.
+      'id' and 'created_at' properties should not be specified when creating subscription as these properties are
+      generated by Nakadi.
+    properties:
+      id:
+        type: string
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+        format: uuid
```

Comment:
I agree with @rcillo that we don't need to provide as less as possible information here. We can use uuid, and can not. It's not client's business. Two things we need to provide - type and maximum length.